pkgs/development/python-modules/pywlroots/default.nix

```diff
@@ -9,7 +9,7 @@
 , pixman
 , pythonOlder
 , udev
-, wlroots
+, wlroots_0_15
```

Comment:
This is inconsistent to the previous changes. I'm fine with either style but IMO it'd be best if we could either use the `wlroots = wlroots_0_x` approach in `all-packages.nix` or this approach here and not mix them. (Not a blocker but would also make reviewing a little bit easier.)

Reply:
This is a python package and needs to be modified `python-packages.nix`, wlroots_0_15 is not part of the python package set, needs to use 'pkgs.wlroots_0_15'

```
 pywlroots = callPackage ../development/python-modules/pywlroots {
    wlroots = pkgs.wlroots_0_15;
  };
```

Reply:
Personally I prefer it explicit. ( ͡° ͜ʖ ͡°)